mmdet/models/detectors/base.py

```diff
@@ -61,6 +61,10 @@ def forward_train(self, imgs, img_metas, **kwargs):
         """
         pass
 
+    @abstractmethod
+    async def async_test(self, img, img_meta, **kwargs):
```

Comment:
maybe renamed to `async_simple_test` if we consider supporting aug test later on?